src/extensions/validation/essayLimitByCharacter.js

```diff
@@ -331,11 +334,17 @@ function setValidationUI(questionInstance, isValid, strLength) {
 function setupEssayValidationUI(questionInstance) {
     const id = questionInstance.getQuestion().response_id;
     const elContainer = document.getElementById(id);
+    if (!elContainer) return;
     const elWordLimit = elContainer.querySelector('.lrn_word_limit');
     const wordLimitText = elWordLimit.textContent;
-    const newWordLimitText = wordLimitText.replace('Word', 'Character');
 
-    elWordLimit.textContent = newWordLimitText;
+    // remove all character except number and "/"
+    const newWordLimitText = wordLimitText.replace(/[^0-9\/ ]*/g, '').trim();
+    if (state.language === 'ja') {
+        elWordLimit.textContent = newWordLimitText + ' 文字制限';
+    } else {
+        elWordLimit.textContent = newWordLimitText + ' Character Limit';
+    }
```

Comment:
I see what you're doing here, nice. I think instead of whitelisting specific languages (in this case English and Japanese) we'll allow the calling page to pass the specific label they want to use to replace "Word".

Reply:
I updated the code 👍